frame/ranked-collective/src/tests.rs

```diff
@@ -308,6 +308,15 @@ fn promote_demote_works() {
 	});
 }
 
+#[test]
+pub fn add_member_to_rank_work(){
+	new_test_ext().execute_with(||{
+		assert_ok!(Club::add_member_to_rank(Origin::root(), 2,1));
```

Comment:
You should be checking the state is updated correctly for the member you are adding to rank